src/store/peginTx/getters.ts

```diff
@@ -52,12 +52,16 @@ export const getters: GetterTree<PegInTxState, RootState> = {
     },
   [constants.PEGIN_TX_GET_REFUND_ADDRESS]: (state: PegInTxState) => {
     let address = '';
-    const coin = EnvironmentAccessorService.getEnvironmentVariables().vueAppCoin;
-    const coinPath = coin === 'main' ? "/0'" : "/1'";
-    // eslint-disable-next-line no-unused-expressions
-    state.addressList?.forEach((walletAddress) => {
-      if (walletAddress.serializedPath === `m/44'${coinPath}/0'/0/0`) address = walletAddress.address;
-    });
+    if (!state.selectedAccount) {
+      address = state.statusInfo.refundAddress;
```

Comment:
I'm assuming that when the screen is transaction status `state.selectedAccount` is not set, then you determine the refund address should come from the `statusInfo` instead. Wouldn't be better to just verify if `statusInfo` is set then use its refund address field?